sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -435,6 +435,18 @@ case class InsertIntoHiveTable(
         logWarning(s"Unable to delete staging directory: $stagingDir.\n" + e)
     }
 
+    //delete the tmpLocation dir
```

Comment:
lint-scala gives error for this line: Insert a space after the start of the comment.